src/chargeTask.hpp

```diff
@@ -22,13 +22,28 @@ class ChargeTask : public Task{
     void init(void);
     /**
      * @brief Charges the device, and exits to CLI on command
-    */
+     * 
+     * @return Current status of charge task
+     */
     STATES_e run(void);
     void exit(void);
 
     private:
+    /**
+     * @brief Buffer for storing command-line input
+     * 
+     * Character array holds input from CLI with a maxiumum size of 'CLI_BUFFER_LEN'
+    */
     char inputBuffer[CLI_BUFFER_LEN];
+    /**
+     * @brief LED status indicator for charging task
+     *
+     * Object provides visual status of device's charging state
+     */
     LEDStatus ledStatus;
+    /**
+     * @brief Timestamp for tracking start of charging process
```

Comment:
When we talk about time, we want to also mention what time is in reference to.  What would a `startTime` of `0` indicate?